Networking/Networking/Remote/RefundsRemote.swift

```diff
@@ -5,53 +5,78 @@ import Alamofire
 ///
 public final class RefundsRemote: Remote {
 
-    /// Retrieves a specific list of `OrderRefunds`s by `orderID`.
+    /// Retrieves all refunds for a specific `orderID`.
     ///
     /// - Parameters:
     ///     - siteID: Site for which we'll fetch remote order refunds.
-    ///     - orderID: Unique identifier for the resource "order" which you are searching
-    ///                 for.
+    ///     - orderID: Unique identifier for the order we're searching for.
     ///     - context: view or edit. Scope under which the request is made;
     ///                determines fields present in response. Default is view.
     ///     - pageNumber: Number of page that should be retrieved.
-    ///     - pageSize: Number of Orders to be retrieved per page.
+    ///     - pageSize: Number of Refunds to be retrieved per page.
     ///     - completion: Closure to be executed upon completion.
     ///
-    public func loadOrderRefunds(for siteID: Int,
-                                 by orderID: Int,
-                                 context: String = Default.context,
-                                 pageNumber: Int = Default.pageNumber,
-                                 pageSize: Int = Default.pageSize,
-                                 completion: @escaping ([OrderRefund]?, Error?) -> Void) {
+    public func loadAllRefunds(for siteID: Int,
+                               by orderID: Int,
+                               context: String = Default.context,
+                               pageNumber: Int = Default.pageNumber,
+                               pageSize: Int = Default.pageSize,
+                               completion: @escaping ([Refund]?, Error?) -> Void) {
         let parameters = [
             ParameterKey.page: String(pageNumber),
             ParameterKey.perPage: String(pageSize),
             ParameterKey.contextKey: context
         ]
         let path = "\(Path.orders)/" + String(orderID) + "/" + "\(Path.refunds)"
         let request = JetpackRequest(wooApiVersion: .mark3, method: .get, siteID: siteID, path: path, parameters: parameters)
-        let mapper = OrderRefundsMapper(siteID: siteID)
+        let mapper = RefundsMapper(siteID: siteID)
+
+        enqueue(request, mapper: mapper, completion: completion)
+    }
+
+    /// Retrieves a single refund by refundID and orderID.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll fetch remote order refunds.
+    ///     - orderID: Unique identifier for the order we're searching for.
+    ///     - refundID: Unique identifier for the refund we're searching for.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func loadRefund(siteID: Int,
+                           orderID: Int,
+                           refundID: Int,
+                           completion: @escaping ([Refund]?, Error?) -> Void) {
```

Comment:
since the goal of this function seems to load a single refund, would it make sense to have the completion block return one optional `Refund` instead of an array of `Refund`s?